.github/workflows/ci.yml

```diff
@@ -33,3 +33,20 @@ jobs:
       run: yarn
     - name: pre fomantic install & gulp build
       run: yarn gulp install
+  codeql:
```

Comment:
moved into `ci.yml` to emit only 1 GH Action workflow instead of 2 separate:

![image](https://user-images.githubusercontent.com/2228672/208111704-f1ddeb32-85a9-4f42-8262-049eb5b2dd2d.png)